flang/lib/Optimizer/Passes/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+add_flang_library(FIRPasses
```

Comment:
I would rather avoid `FIR` here, because these may be `HLFIR` passes as well.  To avoid any confusion, I think, `FlangPasses` might be a better name.

Reply:
Done. I have used `flangPasses` (note the lower case 'f') to be consistent with some of other `libflang` libraries.